jellyfin_kodi/views.py

```diff
@@ -105,29 +105,56 @@
 
 
 def verify_kodi_defaults():
-
     ''' Make sure we have the kodi default folder in place.
     '''
-    node_path = xbmc.translatePath("special://profile/library/video")
-
-    if not os.path.exists(node_path):
-        try:
-            shutil.copytree(
-                src=xbmc.translatePath("special://xbmc/system/library/video"),
-                dst=xbmc.translatePath("special://profile/library/video"))
-        except Exception as error:
-            LOG.warning(error)
-            xbmcvfs.mkdir(node_path)
 
+    source_base_path = xbmc.translatePath("special://xbmc/system/library/video")
+    dest_base_path = xbmc.translatePath("special://profile/library/video")
+
+    # Make sure the files exist in the local profile.
+    # TODO: Investigate why this is needed.
+    # I would think Kodi pulls data from the default profile
+    # if we don't do this.
```

Comment:
When the library is synced, these are what the addon menu/widget use to look up items in the local database.

```
% tree jellyfintvshows767bffe4f11c93ef34b805451a696a4e
jellyfintvshows767bffe4f11c93ef34b805451a696a4e
├── all.xml
├── genres.xml
├── index.xml
├── inprogressepisodes.xml
├── inprogress.xml
├── nextepisodes.xml
├── random.xml
├── recentepisodes.xml
├── recent.xml
└── recommended.xml
```
translates to
![2020-09-03-230518_502x792_scrot](https://user-images.githubusercontent.com/17029228/92195007-f9388980-ee39-11ea-9e94-e46ae804d001.png)

Reply:
The files copied by this function is the following, where a property in `*/index.xml` is overwritten each time.
```
/usr/share/kodi/system/library/video
├── addons.xml
├── files.xml
├── movies
│   ├── actors.xml
│   ├── country.xml
│   ├── directors.xml
│   ├── genres.xml
│   ├── index.xml
│   ├── recentlyaddedmovies.xml
│   ├── sets.xml
│   ├── studios.xml
│   ├── tags.xml
│   ├── titles.xml
│   └── years.xml
├── musicvideos
│   ├── albums.xml
│   ├── artists.xml
│   ├── directors.xml
│   ├── genres.xml
│   ├── index.xml
│   ├── recentlyaddedmusicvideos.xml
│   ├── studios.xml
│   ├── tags.xml
│   ├── titles.xml
│   └── years.xml
├── playlists.xml
└── tvshows
    ├── actors.xml
    ├── genres.xml
    ├── index.xml
    ├── inprogressshows.xml
    ├── recentlyaddedepisodes.xml
    ├── studios.xml
    ├── tags.xml
    ├── titles.xml
    └── years.xml

3 directories, 33 files
```

I have not yet investigated how the jellyfin[type]##### directories are created.